### ISSUE TITLE
bug: bot not working in new Duolingo Layout

### ISSUE THREAD

**Issue by gabrielsozinho** [ChrisRahme/Duolingo_Bot#1]
```cmd
Traceback (most recent call last):
  File "C:\Users\***\***\**\Duolingo_Bot\main.py", line 111, in <module>
    while i < settings['practice_count']:
KeyError: 'practice_count'
```

**Comment by ChrisRahme** [comment]
That's even before starting Duolingo.
Are your `settings.json` and `main.py` in the same folder?
If yes, make sure the JSON structure is correct and the same keys are present.

**Comment by gabrielsozinho** [comment]
@ChrisRahme hey, thanks for answering.

>Are your settings.json and main.py in the same folder?

Yes, they are. My solution was to manually set a number:
```python
while i < 10000:
        print(f'\n>>> Practice {i + 1} of 10000')
```

Even though, I won't use this bot. I think the stories mode gives more xp and is faster. Also, I've already completed all the course.
<br>

I've found a bot that uses stories mode, but it has problems, so I'm gonna create my own bot. Thanks for all!


<sup>sorry for my bad english, I'm still improving
